### PR TITLE
Upgrade to Vite 4

### PR DESCRIPTION
**Pull request by alex-galey**
I don't know how to handle both versions 3 and 4

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "vite-plugin-cloudflare-redirect",
   "description": "Vite plugin to handle Cloudflare's _redirects file",
-  "version": "0.1.0",
+  "version": "0.2.0",
```
Comment (bluwy) [inline package.json]: I don't have permissions to update this branch, can your revert this part, as I'll update it later on release.

File: package.json [hunk 2]
```diff
@@ -32,17 +32,17 @@
     "redirect"
   ],
   "peerDependencies": {
-    "vite": "^3.0.0"
+    "vite": "^4.0.0"
```
Comment (bluwy) [inline package.json]: This can be 
```suggestion
    "vite": "^3.0.0 || ^4.0.0"
```
to support Vite versions.

File: package.json [hunk 3]
```diff
@@ -32,17 +32,17 @@
...
   },
   "peerDependenciesMeta": {
     "vite": {
       "optional": true
     }
   },
   "dependencies": {
-    "cloudflare-redirect-parser": "^0.1.0"
+    "cloudflare-redirect-parser": "0.1.0"
```
Comment (bluwy) [inline package.json]:
```suggestion
    "cloudflare-redirect-parser": "^0.1.0"
```
The caret here should be left as-is.

**Comment by alex-galey** [comment]
no issue with this ?
```js
"devDependencies": {
    "vite": "^4.3.8"
}
```

Could be ? 
```js
  "devDependencies": {
    "vite": "^3.1.4 || ^4.3.8"
```

**Comment by bluwy** [comment]
I don't think we can use that syntax for dev deps, but I'm also fine with testing the latest one locally by default. Thanks for making the changes!

**Review by bluwy — APPROVED** [review]

**Comment by bluwy** [comment]
Released as v0.1.1

**Pull request merged**